app/assets/javascripts/modules/accordion-with-descriptions.js

```diff
@@ -45,6 +47,24 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
       bindToggleForSubsections();
       bindToggleOpenCloseAllButton();
 
+      // When navigating back in browser history to the accordion, the browser will try to be "clever" and return
+      // the user to their previous scroll position. However, since we collapse all but the currently-anchored
+      // subsection, the content length changes and the user is returned to the wrong position (often the footer).
+      // In order to correct this behaviour, as the user leaves the page, we anticipate the correct height we wish the
+      // user to return to by forcibly scrolling them to that height, which becomes the height the browser will return
+      // them to.
+      // If we can't find an element to return them to, then reset the scroll to the top of the page. This handles
+      // the case where the user has expanded all sections, so they are not returned to a particular section, but
+      // still could have scrolled a long way down the page.
+      function storeScrollPosition() {
```

Comment:
Users should be taken back to an open accordion at the right part of the content.